src/DGtal/helpers/Shortcuts.h

```diff
@@ -1761,6 +1758,108 @@ namespace DGtal
         }
 
 
+      /// Outputs a vector field \a vf anchored at \a
+      /// positions as an OBJ file and a material MTL file. Optionnaly
+      /// you can specify the diffuse colors.
+      ///
+      /// @param[in] positions the bases of the vectors of the vector field.
+      /// @param[in] vf the vector field (an array of real vectors).
```

Comment:
```suggestion
      /// @param[in] vf the normal vector field (an array of real vectors).
```

Reply:
as vf is usually used for that

Reply:
It is for displaying *any* vector field. I update the doc.